autotest/utilities/test_gdalalg_raster_proximity.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env pytest
+# -*- coding: utf-8 -*-
+###############################################################################
+# Project:  GDAL/OGR Test Suite
+# Purpose:  'gdal raster proximity' testing
+# Author:   Alessandro Pasotti <elpaso at itopen dot it>
+#
+###############################################################################
+# Copyright (c) 2025, Alessandro Pasotti <elpaso at itopen dot it>
+#
+# SPDX-License-Identifier: MIT
+###############################################################################
+
+import numpy as np
+import pytest
+
+from osgeo import gdal
+
+pytest.importorskip("numpy")
+
+gdal.UseExceptions()
```

Comment:
gdaltest.importorskip_gdal_array(): needed for ReadAsArray()
gdal.UseExceptions() is the default for GDAL pytest

```suggestion
import gdaltest
import numpy as np
import pytest

from osgeo import gdal

gdaltest.importorskip_gdal_array()
pytest.importorskip("numpy")
```